geometry_msgs/PointStamped chest_position
geometry_msgs/Point offset
nav_msgs/Odometry odometry
---
KinovaMoveResponse response
